pkg/designer/domain.go

```diff
@@ -0,0 +1,159 @@
+/*
+ * This file is part of the kubevirt project
```

Comment:
No, it is better to be consistent with existing practice. If a change is desired in this area, it should be done repo wide in a separate commit.